pkgs/applications/virtualization/virtualbox/guest-additions/builder.nix

```diff
@@ -37,13 +37,9 @@ in stdenv.mkDerivation (finalAttrs: {
   '';
 
   patches = [
-    ../gcc-13.patch
-    # https://www.virtualbox.org/changeset/100258/vbox
+    #../gcc-13.patch
```

Comment:
Can't we just get rid of this? The file doesn't exist anymore

Reply:
While I agree this should not be here, I'm not going to make stylistic changes like this in a backport PR. This should be fixed in [master](https://github.com/NixOS/nixpkgs/blob/78ec604bfc86db956e5d46c491f3f34424a9a280/pkgs/applications/virtualization/virtualbox/guest-additions/builder.nix#L39-L43), then backported.